.github/workflows/node.js.yml

```diff
@@ -27,8 +25,8 @@ jobs:
     - name: Install dependencies
       run: npm install
 
-    - name: Run the tests
-      run: npm test
-
-    - name: Build
-      run: npm test && npm run coverage
+    - name: Run the tests and generate coverage report
+      run: npm test -- --coverage
```

Comment:
Is this the right script to run test and coverage? I saw that they are different in `package.json`